nrf_rpc/README.rst

```diff
@@ -3,25 +3,26 @@
 Remote procedure call library (nRF RPC)
 #######################################
 
-nRF RPC is a remote procedure call library for |NCS| enabling inter-processor communication on Nordic Semiconductor SoCs.
-The library is RTOS-agnostic and implements serialization of function calls.
-It is designed to be used with an underlying transport layer, for example `OpenAMP`_.
+nRF RPC is a remote procedure call library for enabling inter-processor communication on Nordic Semiconductor SoCs.
+The library provides a function call serialization framework.
+It is RTOS-agnostic and designed to be used with an underlying transport layer, for example based on `OpenAMP`_.
 
-The nRF RPC library allows for calling a function on remote processors from a local processor, in both synchronous and asynchronous way.
+The nRF RPC library allows for calling a function on a remote processor from a local processor, in both synchronous and asynchronous way.
+It simplifies the serialization of user APIs, such as of a Bluetooth stack, and executing functions implementing those APIs on a remote processor.
 
-Depending on the transport layer, the remote processor is not limited to a single device.
-It can also be a separate device of any type (for example, a PC), or another core on the same system.
+Depending on the transport layer, the local and remote processors are not confined to just one device.
+The remote processor can be located on a separate device of any kind, such as a PC, or it can be another CPU core within the same device.
 
-The nRF RPC library simplifies the serialization of user APIs, such as a Bluetooth stack, and executing of functions implementing those APIs on a remote CPU.
 The library is operating system independent so it can be used with any operating system after porting the OS-dependent layers of the library.
```

Comment:
```suggestion
The library operates system independent, meaning it can be used with any operating system after porting the OS-dependent layers of the library.
```

Reply:
This meant that the library is independent of an operating system, but if maybe we can make it more concise and just tell that "The library can be used with any operating system..."

Reply:
Yes, that would be clearer.